homeassistant/components/tplink/binary_sensor.py

```diff
@@ -39,8 +39,9 @@ class TPLinkBinarySensorEntityDescription(
         key="cloud_connection",
         device_class=BinarySensorDeviceClass.CONNECTIVITY,
     ),
-    # To be replaced & disabled per default by the upcoming update platform.
+    # To be replaced by the upcoming update platform.
     TPLinkBinarySensorEntityDescription(
+        exclude=True,
```

Comment:
We should avoid creating descriptions for entities that will never exist. Instead let's have a set of excluded keys